react/features/toolbox/components/web/Toolbox.js

```diff
@@ -970,6 +972,10 @@ class Toolbox extends Component<Props, State> {
                     text = { _editingDocument
                         ? t('toolbar.documentClose')
                         : t('toolbar.documentOpen') } />,
+            <VideoBlurButton
```

Comment:
Why not move the this._shouldShowButton logic here? I mean something like:
```
this._shouldShowButton('videobackgroundblur') && <VideoBlurButton .... />
```

Reply:
Did not use it that way because i followed SettingsButton below. AbstractButton has a visible property which is set with return value or _shouldShowButton. _shouldShowButton is used in that for OverflowMenuItem which does not have a visible property.